src/inference/gibbs_conditional.jl

```diff
@@ -0,0 +1,149 @@
+struct GibbsConditional{space, C} <: InferenceAlgorithm 
```

Comment:
```suggestion
struct GibbsConditional{space, C<:Distribution} <: InferenceAlgorithm 
```

Reply:
That doesn't work -- the conditional is a function from the conditioned values to a `Distribution`.